### DISCUSSION TITLE
Use .vue extension instead of .md

### DISCUSSION THREAD

**Discussion by joel666** [vuejs/vitepress#2774] [Q&A]
Can I use .vue extension instead of .md for pages ?

**Comment by taozuhong** [comment]
How about `page.vmd`?

**Comment by TechAkayy** [comment]
Yeah, I have a same question. Authoring pages as Vue files makes it easy for us to easily build Static Sites like with vite-ssg/Iles/Astro.

**Reply by TechAkayy** [reply]
Checking on the discord group to see if there are any hacks - https://discord.com/channels/325477692906536972/790509637598314527/1281786291096387687.
